concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -369,7 +369,7 @@ Hale-1973-1798-367	Animals-70	small snake	n	03a.070
 Hale-1973-1798-368	Animals-71	snail	n	03a.071	1543	SNAIL
 Hale-1973-1798-369	Animals-72	shell	n	03a.072	598	SHELL
 Hale-1973-1798-370	Animals-73	scorpion	n	03a.073	1538	SCORPION
-Hale-1973-1798-371	Animals-74	sting	n	03a.074		
+Hale-1973-1798-371	Animals-74	sting	n	03a.074	3798	STINGER
```

Comment:
I think this is correct. Even though the most common meaning would be "a wound or pain caused by or as if by stinging" it looks like in this context, judging by the words before and after in the list, it means "a sharp organ (as of a bee, scorpion, or [stingray](https://www.merriam-webster.com/dictionary/stingray)) that is usually connected with a poison gland or otherwise adapted to wound by piercing and injecting a poison" (https://www.merriam-webster.com/dictionary/sting)